dist/copyright_force.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/python
+
+from subprocess import *
+from sys import *
+from os import rename, remove
+from datetime import *
+
+def pretty_years(s):
```

Comment:
At a glance I have no idea what this function, or any other function in this file does.  Please add high-level docstrings, or rewrite the code to be more self-explanatory.